src/batch-client/index.ts

```diff
@@ -433,6 +436,57 @@ export class ZimbraBatchClient {
 			return ids ? ids.split(',') : [];
 		});
 
+	// For offline Drafts
+	public attach = (files: any, { ...message }) => {
```

Comment:
```suggestion
	public attach = (files: any, message) => {
```

Reply:
is this something we can move to local client class? don't want to put local client related code to api-client

Reply:
Done and moved to zm-x-web